pages/extensions_choices.md

```diff
@@ -39,14 +39,16 @@ Listed below are alternative backends that accept Zipkin format. Some use the sa
     
  - [Apache SkyWalking](https://github.com/apache/incubator-skywalking)
    - When [zipkin-receiver](https://github.com/apache/incubator-skywalking/blob/master/docs/en/setup/backend/backend-receivers.md) is enabled, Skywalking exposes the same HTTP POST endpoints Zipkin does
-     - http port 9411 accepts `/api/v1/spans` (thrift, json) and /api/v2/spans (json, proto) POST requests.
-     - this extension uses the same encoding library and same endpoints as zipkin does.
- - [jaeger](https://github.com/jaegertracing/jaeger)
+     - http port 9411 accepts `/api/v1/spans` (thrift, json) and `/api/v2/spans` (json, proto) POST requests.
+     - this extension uses the same encoding library and same endpoints as Zipkin does.
+ - [Jaeger](https://github.com/jaegertracing/jaeger)
    - When `COLLECTOR_ZIPKIN_HTTP_PORT=9411` is set, Jaeger exposes a partial implementation of Zipkin's HTTP POST endpoints
-     - http port 9411 accepts `/api/v1/spans` (thrift, json) and /api/v2/spans (json, but not proto) POST requests.
-   - When `SPAN_STORAGE_TYPE=kafka` and `zipkin-thrift`, Jaeger reads zipkin v1 thrift encoded span messages from a Kafka topic.
+     - http port 9411 accepts `/api/v1/spans` (thrift, json) and `/api/v2/spans` (json, but not proto) POST requests.
+   - When `SPAN_STORAGE_TYPE=kafka` and `zipkin-thrift`, Jaeger reads Zipkin v1 thrift encoded span messages from a Kafka topic.
      - Note: The above is a [deprecated practice](https://github.com/openzipkin/zipkin/tree/master/zipkin-collector/kafka#legacy-encoding) in Zipkin. Most instrumentation bundle multiple spans per message in v2 format.
-
+ - [Pitchfork](https://github.com/HotelsDotCom/pitchfork)
```

Comment:
@worldtiki probably best if we add a one-line description as a bullet point as it occurs to me not everyone will immediately know what these are :) Want to raise in a separate PR?